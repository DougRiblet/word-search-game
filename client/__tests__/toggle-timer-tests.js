import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ToggleTimer from '../components/Toggle-Timer/toggle-timer-present';

describe('New Game', () => {
  test('Snapshot', () => {
    const dummy = jest.fn();
    const ToggleTimer_tree = renderer.create(
      <ToggleTimer toggleTimer={dummy} allownew={true} timer={false} />
    ).toJSON();
    expect(ToggleTimer_tree).toMatchSnapshot();
  });

  test('renders without crashing', () => {
    const div = document.createElement('div');
    const dummy = jest.fn();
    ReactDOM.render(<ToggleTimer toggleTimer={dummy} allownew={true} timer={false} />, div);
  });

});
