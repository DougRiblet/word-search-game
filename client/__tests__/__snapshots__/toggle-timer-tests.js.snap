// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New Game Snapshot 1`] = `
<div
  id="toggletimer"
>
  <div
    id="togglecheck"
  >
    <label
      htmlFor="timer-choice"
    >
      <input
        checked={false}
        disabled={false}
        id="timer-choice"
        name="timer-choice"
        onChange={[MockFunction]}
        type="checkbox"
      />
    </label>
  </div>
</div>
`;
